### DISCUSSION TITLE
Is there any way to save contents to html and keep original format from a worksheet?

### DISCUSSION THREAD

**Discussion by dsu0629** [ClosedXML/ClosedXML#2215] [Q&A]
I want to copy the contents of a worksheet to an email body(HTML format) and keep its original format. Is there a way to do it?

**Comment by agrath** [comment]
@dsu0629 no it does not seem possible. You could write a converter that iterated over the sheet cells but then you'd have to apply all the style rules including things like merging cells. Seems like it would be very fiddly.
